db/system_keyspace.cc

```diff
@@ -256,6 +256,7 @@ schema_ptr system_keyspace::topology() {
             .with_column("num_tokens", int32_type)
             .with_column("shard_count", int32_type)
             .with_column("ignore_msb", int32_type)
+            .with_column("supported_features", utf8_type)
```

Comment:
@kbr-scylla I think it goes hand in hand how CQL statements are built to enable/disable features. If we use set<text> and use + operator to add features to the set, we can make the end result eventually consistent, so there is an advantage. There is also a guarantee that the feature set only contains unique features at all steps. Looks like a separate follow up work, since for the transitioning commit as it is now, it is easier to *move* features to raft as is, and then  change the way htey are represented in raft. I am concerned with the scope of this series.

Reply:
Eventually consistent? Isn't it managed by Raft?

I don't think there's a difference in terms of correctness (since we know how to implement sets in C++), but it's much nicer to have a set. It's also better to start with a set, incremental development and schema changes don't go together well.